source/_docs/alternate-docroot.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Create an Alternate Docroot
+description: Learn how to create an alternate docroot to serve your Pantheon site from.
+categories: [developing,sites]
+tags: [platform,code]
+keywords: Composer, docroot
+---
+
```

Comment:
@ataylorme can you add a short intro with example use case(s) for serving sites from an alternate docroot?